components/Storyblok/SbTypeform.tsx

```diff
@@ -25,10 +26,11 @@ export type SbTypeformProps = {
     transitiveSearchParams: string;
     displayAsFullScreenModal: boolean;
     transparent: boolean;
+    paddingTop: PaddingType;
+    paddingBottom: PaddingType;
```

Comment:
I've added padding options to make easier to use in the grid component.

Reply:
Thanks - after we merge this in, I'll think more about whether we need this. Will do a clean up at the end 😄